template/base/_.gitignore

```diff
@@ -14,3 +15,5 @@ node_modules*/
 npm-debug.log
 package-lock.json
 yarn-error.log
+# end managed by skuba
+
```

Comment:
We need to be consistent with file endings here and/or reinstate the sanitisation logic. Looks like there's an extra newline here, trailing space in `_.eslintignore`, and no trailing newline in `_.dockerignore`